stdlib/public/SwiftShims/RefCount.h

```diff
@@ -274,22 +271,22 @@ struct RefCountBitOffsets<8> {
 // 32-bit inline
 template <>
 struct RefCountBitOffsets<4> {
-  static const size_t ReservedShift = 0;
-  static const size_t ReservedBitCount = 0;
-  static const uint32_t ReservedMask = maskForField(Reserved);
-
-  static const size_t UnownedRefCountShift = shiftAfterField(Reserved);
-  static const size_t UnownedRefCountBitCount = 8;
+  static const size_t IsImmortalShift = 0;
+  static const size_t IsImmortalBitCount = 1;
+  static const uint64_t IsImmortalMask = maskForField(IsImmortal);
+  
+  static const size_t UnownedRefCountShift = shiftAfterField(IsImmortal);
+  static const size_t UnownedRefCountBitCount = 7;
   static const uint32_t UnownedRefCountMask = maskForField(UnownedRefCount);
 
   static const size_t IsDeinitingShift = shiftAfterField(UnownedRefCount);
   static const size_t IsDeinitingBitCount = 1;
   static const uint32_t IsDeinitingMask = maskForField(IsDeiniting);
-
-  static const size_t StrongExtraRefCountShift = shiftAfterField(IsDeiniting);
+  
+  static const size_t StrongExtraRefCountShift = shiftAfterField(IsImmortal);
```

Comment:
I think this change will make StrongExtraRefCount overlap UnownedRefCount, and isn't needed.

Reply:
Aha, didn't know about --long-test, thank you!

And yes, that's a bug, good catch

Reply:
I *suspect* that `--long-test` would have caught this. Not sure. Obviously, feel free to add something that would.

Reply:
I think the main reason I didn't catch it is that that part is 32 bit only

Reply:
That would do it. Make sure to test with `--ios` to test 32-bit, or `--watchos` if you don't want to install an old simulator.